enhancements/sdk-kubernetes-1.19.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Support Kubernetes 1.19 in Operator SDK
+authors:
+  - "@joelanford"
+reviewers:
+  - "@estroz"
+  - "@camilamacedo86"
+  - "@jmrodri"
+approvers:
+  - "@hasbro17"
+  - "@jmccormick2001"
+  - "@jmrodri"
+creation-date: 2020-05-21
+last-updated: 2020-05-21
+status: provisional
+see-also:
+  - "https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19"
+replaces: []
+superseded-by: []
+---
+
+# Support Kubernetes 1.19 in Operator SDK
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] ~~Graduation criteria for dev preview, tech preview, GA~~ (N/A)
+
+## Open Questions
+
+1. controller-runtime currently uses a released kubebuilder version to get the necessary tools to
+   run envtest (kube-apiserver and etcd). This causes a chicken-and-egg problem because kubebuilder
```

Comment:
These [binaries](https://storage.googleapis.com/kubebuilder-tools) can be pushed independently of a kubebuilder release though right? I'm uncertain whether they're created manually or via CI, but it seems we can push a binary set for a greater k8s version than kubebuilder supports.

Reply:
I'll leave this in as a reminder, but I agree this is out-of-scope. After all, previous releases of controller-runtime have had this issue but that didn't stop them from successfully bumping k8s versions.